lighthouse-treemap/types/treemap.d.ts

```diff
@@ -45,5 +48,3 @@ declare global {
     signal?: AbortSignal;
   }
 }
-
-export {};
```

Comment:
what's the force module situation these days such that this isn't necessary? I feel like I remember reading a PR description or comment about this recently, but couldn't find it quickly.

Reply:
> what's the force module situation these days such that this isn't necessary? I feel like I remember reading a PR description or comment about this recently, but couldn't find it quickly.

it's kind of dumb: if there's any `import` or `export` statements it's a module (which is why we had the empty ones in there before), and if you want an explicit `declare global {}` block in there it has to be a module, otherwise it's ambient and _everything_ is a global type.

Fortunately(?) we almost always `import` statements these days, and the error message is a lot better, basically saying you can't `declare global` unless this file is a module.

I vaguely recall a few years back in the es modules transition wars the idea of using that same heuristic to automatically detect if a file was an `.mjs` vs `.cjs` file without a file extension/package.json `type` entry, and then everyone realized that would be super fragile and annoying and decided not to go that way. But here is it anyways :)